### DISCUSSION TITLE
意识到一个有趣的问题

### DISCUSSION THREAD

**Discussion by TAKO-COLO** [share121/inter-knot#2824] [其他]
就目前来看这个网站是没有审核的，那有没有可能会有人在这里发本子和学习资料（？）

**Comment by fzdqz** [comment]
这话说的<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >不得不拿学习资料出来分享了
美好知识的传送入口，懂得都懂<img src="https://bbs-static.miyoushe.com/static/2024/09/10/d60393fe879038382e5c7cdcf7ca25fd_7267933421783671773.png" alt="飞霄 打架" width="75" >不用谢我，我是个好人 https://www.bilibili.com/read/readlist/rl330029
 https://www.bilibili.com/read/readlist/rl331197
 https://www.bilibili.com/read/readlist/rl334156
 https://www.bilibili.com/read/readlist/rl336027
 https://www.bilibili.com/read/readlist/rl336028

**Reply by KELERSAMA** [reply]
Bilibili出现时就不简单了

**Reply by zdsy03** [reply]
我狠狠的学，忘情的学！

**Comment by ChenRei000** [comment]
哈人捏，小心被举办
